### PR TITLE
settings are set in config file

### LINKED ISSUE #33: Check if all setting set via env variables can be set in config file
It seems, `INFLUX_PASSWORD` is env-var-only same is `AWX_AUTH_TOKEN`

### PR DESCRIPTION
**Pull request by Polina-Gubina**
Resolve #33 
Influx password is set in configuration file

### CODE REVIEW

File: delatore/configuration.py [hunk 1]
```diff
@@ -27,10 +28,12 @@ def read_config(config_file=RESOURCES['config.ini']):
     if defaults:
         token = defaults.get('token', os.getenv('token'))
         chat_id = defaults.get('chat_id', os.getenv('chat_id'))
-        return BotConfig(token, chat_id)
+        influx_password = defaults.get('influx_password', os.getenv('influx_password'))
```
Comment (outcatcher) [inline delatore/configuration.py]: `influx_password` is incorrect evn var name, should be `INFLUX_PASSWORD`

File: delatore/configuration.py [hunk 2]
```diff
@@ -27,10 +28,12 @@ def read_config(config_file=RESOURCES['config.ini']):
...
+        return BotConfig(token, chat_id, influx_password)
     return BotConfig(
         os.getenv('token'),
         os.getenv('chat_id'),
+        os.getenv('influx_password')
```
Comment (outcatcher) [inline delatore/configuration.py]: should be `INFLUX_PASSWORD`

File: delatore/__main__.py [hunk 3]
```diff
@@ -10,11 +10,13 @@ def _config():
     arg_p.add_argument("--config", default=None, help="Configuration file to use")
     arg_p.add_argument("--chat", default=os.getenv("chat_id"), help="Chat for notifications")
     arg_p.add_argument("--token", default=os.getenv("token"), help="Telegram bot token")
+    arg_p.add_argument("--influx_password", default=os.getenv("INFLUX_PASSWORD"), help="Influx password")
+    arg_p.add_argument("--awx_auth_token", default=os.getenv("AWX_AUTH_TOKEN"), help="AWX_AUTH_TOKEN")
```
Comment (anton-sidelnikov) [inline delatore/__main__.py]: help string can be like: "OAuth2 Token for Ansible Tower"

File: delatore/sources/awx_api.py [hunk 4]
```diff
@@ -35,7 +36,7 @@ class AWXApiClient:
 
     def __init__(self):
         self.session = requests.session()
-        self.session.headers.update({'Authorization': f'Bearer {os.getenv("AWX_AUTH_TOKEN")}'})
+        self.session.headers.update({'Authorization': f"{BOT_CONFIG.AWX_AUTH_TOKEN}"})
```
Comment (outcatcher) [inline delatore/sources/awx_api.py]: `Bearer ` part should remain

**Pull request merged**